app/views/rooms/show.html.erb

```diff
@@ -0,0 +1,124 @@
+<div>
+  <%= "Home/#{@room.room_hash}"%>
+</div>
+
+<% if @current_jam %>
+  <%= react_component("waveform_player", { id: @current_jam.id, audioUrl: url_for(@current_jam.file) }) %>
+  
+  <div id="current-jam-attributes">
+    <ul>
+      <li>Latest JAM <%= @current_jam.created_at.strftime("%B, %d %Y") %></li>
+      <li>FILE: <%= @current_jam.filename %></li>
+      <li>BPM: <%= @current_jam.bpm %></li>
+    </ul>
+  </div>
+<% else %>
+  <div>
+    This room is brand new! Upload a track to get started!
+  </div>
+<% end %>
+
+<div id="previous-jams">
+  <% unless @jams.empty? %>
+    Previous JAMs on this track
+    <% @jams.each do |jam| %>
+      <%= react_component("waveform_player", { id: jam.id, audioUrl: url_for(jam.file) }) %>
+      <div id="previous-jam-attributes">
+        <ul>
+          <li>Latest JAM <%= jam.created_at.strftime("%B, %d %Y") %></li>
+          <li>FILE: <%= jam.filename %></li>
+          <li>BPM: <%= jam.bpm %></li>
+          <li><%= link_to("Download Track", url_for(jam.file), class: "button") %>  </li>
+        </ul>
+      <div>
+    <% end %>
+    <div id="previous-jams-react-element"></div>
+  <% end %>
+</div>
+
+<div>
+  <% if @current_jam %>
+    <h1>JAM on this track!</h1>
+    <%= link_to("Download Track", url_for(@current_jam.file), class: "button") %>  
+  <% end %>
+
+  <button id="upload-jam-button" class="button">Upload New Track</button>
+</div>
+
+<div class="modal">
+  <div class="modal-background"></div>
+  <div class="modal-card">
+    <%= form_with(model: @room, url: room_path(@room.room_hash), :multipart => true,  local: true) do |form| %>
```

Comment:
you don't want to mix different styles so `multipart: true` vs `:multipart => true`